dev-vcs/gitqlient/gitqlient-1.6.2.recipe

```diff
@@ -58,13 +42,15 @@ REQUIRES="
 	lib:libQt5Core$secondaryArchSuffix
 	lib:libQt5Gui$secondaryArchSuffix
 	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
```

Comment:
```suggestion
	lib:libQt5Gui$secondaryArchSuffix
	lib:libQt5Network$secondaryArchSuffix
	lib:libQt5Widgets$secondaryArchSuffix
```

Reply:
Fixed. Out of curiosity, does this order matter?